packages/icons/test/index.spec.js

```diff
@@ -14,3 +15,16 @@ describe('Icons render test : ', () => {
     });
   });
 });
+
+// RENDER TEST
+describe('Sprite Icons render test : ', () => {
```

Comment:
there's a built-in way to navigate through tests, instead of `forEach` on keys of the icons, you can use `describe.each`.
https://jestjs.io/docs/en/api#describeeachtablename-fn-timeout

Reply:
yeap. fixed